client/src/components/forms/HostGameForm.tsx

```diff
@@ -3,54 +3,40 @@ import { useNavigate } from 'react-router-dom';
 import { GamesContext, VenuesContext } from '../../App';
 import { createGame } from '../../services/apiService';
 import NavBar from '../common/NavBar';
+import { Venue } from '../../@types/venue';
+import { Game } from '../../@types/game';
 
 function HostGameForm() {
   const navigate = useNavigate()
-  const venues = useContext(VenuesContext)
+  const venues: Venue[] = useContext(VenuesContext).venues;

```

Comment:
Looks like you destructured it like this `const { venues } = useContext(VenuesContext)` everywhere else. Any reason you do it differently here?